rotkehlchen/api/server.py

```diff
@@ -336,6 +337,7 @@
     ('/calendar', CalendarResource),
     ('/calendar/reminders', CalendarRemindersResource),
     ('/calendar/google', GoogleCalendarResource),
+    ('/external_services/monerium', MoneriumOAuthResource),
```

Comment:
```suggestion
    ('/services/monerium', MoneriumOAuthResource),
```
also works but it is a nitpick